lib/node_modules/@stdlib/math/base/special/sinpi/src/main.c

```diff
@@ -0,0 +1,81 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/*
+* Notes:
+*	=> sin(-x) = -sin(x)
+*	=> sin(+n) = +0, where `n` is a positive integer
+*	=> sin(-n) = -sin(+n) = -0, where `n` is a positive integer
+*	=> cos(-x) = cos(x)
```

Comment:
I realize it's the same in the JS code, but we don't use this style anywhere else aside from `cospi` and `sinpi`. I propose we change this comment to:

```suggestion
* Notes:
*
* -   sin(-x) = -sin(x)
* -   sin(+n) = +0, where `n` is a positive integer
* -   sin(-n) = -sin(+n) = -0, where `n` is a positive integer
* -   cos(-x) = cos(x)
```

Reply:
cc @kgryte

Reply:
Agreed. Let's also move this to the Doxygen block at L37.

Reply:
Instead of `Notes:`, move to under a `## Notes` section. This applies to the JS file, as well.

Reply:
I've made the changes.